path_config.go

```diff
@@ -76,8 +76,22 @@ func (b *backend) configFields() map[string]*framework.FieldSchema {
 }
 
 func (b *backend) configCreateUpdateOperation(ctx context.Context, req *logical.Request, fieldData *framework.FieldData) (*logical.Response, error) {
+	conf, err := readConfig(ctx, req.Storage)
```

Comment:
This API didn't respect prior values set for updates. For example, if a user provided some parameter on create and didn't provide the same parameter on update, its prior value (from the create) would not be retained. I fixed this to operate similar to most other Vault APIs.

Reply:
Should this be considered its own bugfix and get backported? Sounds like a past oversight to me maybe.

Reply:
Ya, that a good consideration. I'm actually going to remove other isolated bug fixes in this PR to new PRs so that they can be backported.

Reply:
I've split this out into a separate bug fix PR in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/43. I'll remove the change from this PR as well. Thanks, @robmonte!